test/functional/test_framework/crypto/bip324_cipher.py

```diff
@@ -25,6 +25,8 @@ def pad16(x):
 
 def aead_chacha20_poly1305_encrypt(key, nonce, aad, plaintext):
     """Encrypt a plaintext using ChaCha20Poly1305."""
+    if plaintext is None:
```

Comment:
29390: also a part of #6330 and it probably belongs there and should be dropped here

Reply:
done